projects/storefrontlib/src/cms-components/checkout/components/shipping-address/shipping-address.component.ts

```diff
@@ -57,17 +41,12 @@ export class ShippingAddressComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit() {
-    this.goTo = null;
-    this.checkoutStepUrlNext = this.checkoutConfigService.getNextCheckoutStepUrl(
-      this.activatedRoute
-    );
-    this.checkoutStepUrlPrevious = 'cart';
-
     this.isLoading$ = this.userAddressService.getAddressesLoading();
     this.existingAddresses$ = this.userAddressService.getAddresses();
```

Comment:
Correct me if I'm wrong, but I don't see any place where this one is unsubscribed on component destroy?

Reply:
Observables here are not subscribed to and cards$ is unsubscribed in html template by `| async` pipe.